Automattic/woocommerce-payments
# Show total balance (pending + available) instead of pending balance

Fixes https://github.com/Automattic/woocommerce-payments/issues/8070

#### Changes proposed in this Pull Request

- Show total balance and available balance instead of available balance and pending balance.

Before:
<img width="565" alt="Screenshot 2024-02-07 at 07 50 22" src="https://github.com/Automattic/woocommerce-payments/assets/73803630/c13afe9a-edcb-4d44-8334-0ba872d12374">

After:
<img width="549" alt="Screenshot 2024-02-07 at 07 51 36" src="https://github.com/Automattic/woocommerce-payments/assets/73803630/6b4dfd91-379b-4bcd-8144-6fbd5a9e6fa0">

Total balance will show tooltip if it is negative.

<img width="653" alt="Screenshot 2024-02-07 at 07 34 50" src="https://github.com/Automattic/woocommerce-payments/assets/73803630/56a08a2c-ae6d-46e3-9dbb-3f5c83f434a2">

#### Testing instructions

* Open Payments > Overview page.
* With base branch, you will see available and pending balance.
* With PR branch, you will see total and available balance.
* Hard-code available and pending balance like how I accidentally committed here: https://github.com/Automattic/woocommerce-payments/pull/8140/commits/3c3373613c69032c4664d164d26f871d708b3c01. Set the value so total balance is negative, expects a tooltip next to total balance.

-------------------


**Post merge**
